packages/fxa-react/lib/interfaces.ts

```diff
@@ -0,0 +1,5 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+export type GleanDataType2fa = 'setup' | 'inline setup' | 'replace';
```

Comment:
I'm not afraid of spaces, but I also don't know how Glean is going to handle them.

Reply:
Hmmm, well I'm certainly open to using a dash or something instead.